nyaa/static/css/main.css

```diff
@@ -1,3 +1,14 @@
+@import url(//fonts.googleapis.com/earlyaccess/notosansjapanese.css);
+@import url('https://fonts.googleapis.com/css?family=Noto+Sans');
```

Comment:
Since http://nyaa.si always redirects to https://nyaa.si,
I think you can drop the `https:` like on the first URL.